### PR TITLE
Add: background style

### PR DESCRIPTION
**Pull request by nojiritakeshi**
The following styles were added.
```
bgImage: "background-image",
bgPosition: "background-position",
bgPositionX: "background-position-x",
bgPositionY: "background-position-y",
bgSize: "background-size",
bgRepeat: "background-repeat",
bgAttachment: "background-attachment",
bgClip: "background-clip",
bgOrigin: "background-origin",
```

### CODE REVIEW

File: packages/system/src/background.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+       */
+      bgImage: CSSValue<"backgroundImage"> | AutoPrefix;
+      /**
+       * @see backgroundPosition
+       */
+      bgPosition: CSSValue<"backgroundPosition"> | AutoPrefix;
+      /**
+       * @see backgroundPositionX
+       */
+      bgPositionX: CSSValue<"backgroundPositionX"> | AutoPrefix;
```
Comment (kotarella1110) [inline packages/system/src/background.ts:24]:
```suggestion
      bgPositionX: CSSValue<"backgroundPositionX", true> | AutoPrefix;
```

**Review by kotarella1110 — COMMENTED** [review]
@nojiritakeshi 
Thank you for the PR 😚
Since `positionX` and `positionY` can accept px values, I believe they should allow the number type as well.
I have made the suggestion, so please make the necessary changes.
Also, it would be great to have tests for these properties.

File: packages/system/src/background.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      /**
+       * @see backgroundPositionY
+       */
+      bgPositionY: CSSValue<"backgroundPositionY"> | AutoPrefix;
```
[the remark this answers is not in the document]
Comment (nojiritakeshi) [reply packages/system/src/background.ts:28]: @kotarella1110 
Adding to this point, is there any problem with modifying the background-size as well?

[the remark this answers is not in the document]
Comment (kotarella1110) [reply packages/system/src/background.ts:28]: @nojiritakeshi Yes! Thank you 👍

**Comment by nojiritakeshi** [comment]
@kotarella1110 @poteboy 
I have corrected the points you pointed out! Please check it.

**Review by kotarella1110 — APPROVED** [review]
LGTM 🎉

**Review by poteboy — APPROVED** [review]
Thank you :)

**Comment by masa5555** [comment]
#150

**Pull request merged**